e2e/docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM envoyproxy/envoy:v1.18.3 AS envoy
+WORKDIR /app
+RUN cp $(which envoy) ./envoy
+
+FROM golang:1.16 AS go
+
+FROM tinygo/tinygo:0.18.0
+ENV PATH $PATH:/usr/local/go/bin
+RUN apt-get clean && \
+    apt-get update && \
+    apt-get install -y make
+COPY --from=envoy /app/envoy /usr/local/bin
```

Comment:
why not directly copy from /usr/local/bin/envoy?

Reply:
It may be broken if the binary's path changed inside the image. We can determine where the binary is regardless of where it is.